cpmpy/solvers/minizinc.py

```diff
@@ -135,6 +135,7 @@ def __init__(self, cpm_model=None, subsolver=None):
         self.mzn_model.add_string("% Generated by CPMpy\ninclude \"globals.mzn\";\n\n")
         # Prepare solve statement, so it can be overwritten on demand
         self.mzn_txt_solve = "solve satisfy;"
+        self.mzn_result = None
```

Comment:
the docstring of minizinc interface needs to be updated to contain this new element, like the one of or-tool has, no?

Reply:
yes, updated it, but now i see that minizinc is not even in our list of solver api's and i don't know why..

Reply:
yes, it should be. Is there another missing?

Reply:
it wasn't in the init, fixed that, now trying to figure out linebreaks :(